pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -56,7 +56,8 @@ type PipelineRunFacts struct {
 	// needed, via the `Skip` method in pipelinerunresolution.go
 	// The skip data is sensitive to changes in the state. The ResetSkippedCache method
 	// can be used to clean the cache and force re-computation when needed.
-	SkipCache map[string]bool
+	SkipCache                  map[string]bool
+	ScopeWhenExpressionsToTask bool
```

Comment:
the docstring for PipelineRunFacts could use an update i think as well since it seems to be trying to list everything in this object

might be better to replace it with a description of what this object seems to be for instead of a list of its contents, e.g. something like "PipelineRunFacts holds the state of all of the components that make up the Pipeline graph and can be used to <something something christie got lazy 😅 >"

Reply:
updated the docstring to explain its purpose